agent/checks/os_service_windows.go

```diff
@@ -0,0 +1,114 @@
+//go:build windows
+// +build windows
+
+package checks
+
+import (
+	"fmt"
+	"syscall"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+var (
+	win windowsSystem = windowsOS{}
+)
+
+type OSServiceClient struct {
+	scHandle windows.Handle
+}
+
+func NewOSServiceClient() (*OSServiceClient, error) {
+	var s *uint16
+	scHandle, err := win.OpenSCManager(s, nil, windows.SC_MANAGER_CONNECT)
```

Comment:
My guess is that this should work as well
```suggestion
	scHandle, err := win.OpenSCManager(nil, nil, windows.SC_MANAGER_CONNECT)
```